src/library/transform/scan.cpp

```diff
@@ -1,147 +1,64 @@
-/* ************************************************************************
```

Comment:
AMD is also the copyright holder on the Bolt library, also under Apache, so the boilerplate was correct.